ui/webui/src/components/app.jsx

```diff
@@ -69,6 +69,10 @@ export const Application = () => {
             buildstamp => setBeta(!getIsFinal(buildstamp)),
             ex => console.error("Failed to parse anaconda buildstamp file")
         );
+
+        getOsReleaseByKey("PRETTY_NAME").then(
+            setIsPrettyName,
```

Comment:
Please remove comma, this is just not failing because - JS - but it does not belong there.